File: app/src/main/java/com/cna/mineru/cna/Utils/WebViewActivity.java
package com.cna.mineru.cna.Utils;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import com.cna.mineru.cna.R;

public class WebViewActivity extends AppCompatActivity {
    private WebView mWebView;
    private String myUrl;

    @SuppressLint("SetJavaScriptEnabled")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_webview);
        myUrl = getString(R.string.web_ip_set)+"/policy";
        String url = getIntent().getStringExtra("value");

        mWebView = (WebView) findViewById(R.id.webview);
        mWebView.setWebViewClient(new WebViewClient());
        WebSettings webSettings = mWebView.getSettings();
        webSettings.setJavaScriptEnabled(true);
        webSettings.setSupportZoom(true);
        webSettings.setBuiltInZoomControls(true);
        mWebView.loadUrl(myUrl + "/"+ url);

    }
}

File: app/src/main/java/com/cna/mineru/cna/Utils/WebViewActivity.kt
package com.cna.mineru.cna.Utils

import android.annotation.SuppressLint
import android.os.Bundle
import android.webkit.WebView
import android.webkit.WebViewClient
import androidx.appcompat.app.AppCompatActivity

import com.cna.mineru.cna.R

class WebViewActivity : AppCompatActivity() {
    private var mWebView: WebView? = null
    private var myUrl: String? = null

    @SuppressLint("SetJavaScriptEnabled")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_webview)
        myUrl = getString(R.string.ip_set) + "/policy"
        val url = getIntent().getStringExtra("value")

        mWebView = findViewById(R.id.webview) as WebView
        mWebView!!.webViewClient = WebViewClient()
        val webSettings = mWebView!!.settings
        webSettings.javaScriptEnabled = true
        webSettings.setSupportZoom(true)
        webSettings.builtInZoomControls = true
        mWebView!!.loadUrl("$myUrl/$url") // 접속 URL

    }
}
